### ISSUE TITLE
ironframework.io serves malicious content

### ISSUE THREAD

**Issue by austinjones** [iron/iron#632]
The owners of the `ironframework.io` domain are serving malicious content.  Could the documentation links in the readme and the latest crates.io release be updated?

**Comment by pietroalbini** [comment]
Hello! I'm Pietro from the crates.io team!

We're making a change to crates.io to remove ironframework.io links from the crates.io web UI, and documentation links pointing to it will be replaced with docs.rs links automatically. If you gain control of the domain back please email us at help@crates.io and we'll happily revert the domain block!

Note that this does not affect the availability of the crates in any way.

**Comment by phlmn** [comment]
@reem Can you remove the website field from this repository? I do not have the permissions to do so.

**Comment by theptrk** [comment]
This is true, do not visit that link. I'll see what I can do to fix this.

**Comment by reem** [comment]
I've removed it. Sorry about that, the registration lapsed. Please let me know if there is anywhere else it can be removed.

**Comment by theptrk** [comment]
Thanks @reem. Ironwebframework.com is available if you want to point to a new domain.
